### PR TITLE
upgrades: innodb upgrade

### PR DESCRIPTION
**Pull request by jalavik**
* Backports 50b04b8 from Invenio that adds the InnoDB upgrade
  script.

Signed-off-by: Jan Aage Lavik <jan.age.lavik@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
@jalavik we should backport also inveniosoftware/invenio@66ad8455f25d0c8ac13700c0437064153fc7ca0a. Are you planning to remove `invenio.modules.oaiharverster` module?

**Comment by jirikuncar** [comment]
@jalavik please mention the original commit as `inveniosoftware/invenio@<sha1>` or keep the original commit message.

**Comment by jalavik** [comment]
@jirikuncar inveniosoftware/invenio@66ad845 is sorted already in #3. We should remove it yes, but let's wait for #3 and then we can release `0.1`.

I updated the commit message.

**Comment by jirikuncar** [comment]
it's usually good idea to release initial version with minimal changes from original Invenio module.

**Comment by jalavik** [comment]
Make sense. I can port those few lines then from inveniosoftware/invenio@66ad845 and prepare a release.

**Comment by jalavik** [comment]
@jirikuncar I think this should be ready to go now. Some tests won't pass due to missing Invenio installation (still a `FIXME` for all external modules I guess?). 

We can release like this although the module is not in a great state due to missing daemon and GUI, so we'd want to release another version with the new goodies ASAP after initial release.

File: .travis.yml [hunk 1]
```diff
@@ -35,16 +37,24 @@ cache:
...
+  - REQUIREMENTS=release REXTRAS=docs,tests
+  - REQUIREMENTS=devel REXTRAS=docs,tests
 
 python:
   - "2.7"
 
+addons:
+  apt:
+    packages:
+      - apache2
```
Comment (jirikuncar) [inline .travis.yml]: do we need apache2? (can you verify the others too?)

File: .travis.yml [hunk 2]
```diff
@@ -35,16 +37,20 @@ cache:
...
 
 python:
   - "2.7"
 
+addons:
+  apt:
+    packages:
+      - git
+
+
```
Comment (jirikuncar) [inline .travis.yml]:
```diff
-
```

**Pull request merged**